sdk/lib/dmilib/dmilib.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * PROJECT:     ReactOS DMI/SMBIOS Library
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dmilib.h
+ * PURPOSE:     SMBIOS table parsing functions
+ * PROGRAMMERS: Timo Kreuzer (timo.kreuzer@reactos.org)
+ * REFERENCES:  - http://www.dmtf.org/sites/default/files/standards/documents/DSP0134_3.0.0.pdf
+ *              -
+ */
+
+#pragma once
+
+enum _ID_STRINGS
+{
+    ID_NONE = 0,
+    BIOS_VENDOR,
+    BIOS_VERSION,
+    BIOS_DATE,
+    SYS_VENDOR,
+    SYS_PRODUCT,
+    SYS_VERSION,
+    SYS_SERIAL,
+    BOARD_VENDOR,
+    BOARD_NAME,
+    BOARD_VERSION,
+    BOARD_SERIAL,
+    BOARD_ASSET_TAG,
+
+    ID_STRINGS_MAX,
+};
+
+VOID
+ParseSMBiosTables(
+    _In_reads_bytes_(TableSize) PVOID SMBiosTables,
+    _In_ ULONG TableSize,
+    _Inout_ PCHAR * Strings);
```

Comment:
Instead of the `_Inout_` notation, you can use: `_Inout_updates_(ID_STRINGS_MAX)`.
This indicates that the array should be of the size of `ID_STRINGS_MAX`